x/feesplit/msg_convert.go

```diff
@@ -0,0 +1,74 @@
+package feesplit
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/okex/exchain/libs/cosmos-sdk/baseapp"
+	sdk "github.com/okex/exchain/libs/cosmos-sdk/types"
+	"github.com/okex/exchain/x/common"
+	"github.com/okex/exchain/x/feesplit/types"
+)
+
+var (
+	ErrCheckSignerFail = errors.New("check signer fail")
+)
+
+func init() {
+	RegisterConvert()
+}
+
+func RegisterConvert() {
+	enableHeight := int64(0)
```

Comment:
For a new msg，does enableHeight same to the new milestone height?

Reply:
> For a new msg，does enableHeight same to the new milestone height?

yes,  most of case is same to milestone, different msg maybe have different milestone, same module different msg maybe have different milestone